### PR TITLE
update: gradle metadata publishing

### PR DESCRIPTION
**Pull request by katia-energizer**
Updated metadata publishing for newer Gradle versions; removed Experimental notion.

### CODE REVIEW

File: pages/docs/reference/building-mpp-with-gradle.md [hunk 1]
```diff
@@ -1295,23 +1295,22 @@ kotlin {
...
-In Gradle 5.3 and above, the module metadata is always used during dependency resolution, but publications don't 
-include any module metadata by default. To enable module metadata publishing, add 
-`enableFeaturePreview("GRADLE_METADATA")` to the root project's `settings.gradle` file. With older Gradle versions, 
-this is also required for module metadata consumption.
+In Gradle 6.0 and above, the module metadata is always used during dependency resolution and included in publications.
 
-> Note that the module metadata published by Gradle 5.3 and above cannot be read by Gradle versions older
-> than 5.3. 
-{:.note}
+In Gradle 5.3 and above, the module metadata is used during dependency resolution, but publications don't
```
Comment (h0tk3y) [inline pages/docs/reference/building-mpp-with-gradle.md:1309]: Maybe replace `5.3 and above` with `versions below 6.0` or `versions 5.3 to 5.6.4`?

File: pages/docs/reference/building-mpp-with-gradle.md [hunk 2]
```diff
@@ -962,7 +962,7 @@ kotlin {
 </div>
 </div>
 
-Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#experimental-metadata-publishing-mode) and the project 
+Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#metadata-publishing) and the project
```
Comment (h0tk3y) [inline pages/docs/reference/building-mpp-with-gradle.md:965]: Remove `experimental` from this line, too? Or even rephrase it as `if a multiplatform library is [published with Gradle module metadata](link)`?

File: pages/docs/reference/building-mpp-with-gradle.md [hunk 3]
```diff
@@ -962,7 +962,7 @@ kotlin {
 </div>
 </div>
 
-Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#experimental-metadata-publishing-mode) and the project 
+Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#metadata-publishing) and the project 
 is set up to consume the metadata as well, then it is enough to specify a dependency only once, for the common source set.
```
Comment (h0tk3y) [inline pages/docs/reference/building-mpp-with-gradle.md:966]: `is set up to consume the metadata` – always true with Gradle 5.3+, maybe remove?

File: pages/docs/reference/building-mpp-with-gradle.md (hunk 1 above)
Comment (katia-energizer) [reply pages/docs/reference/building-mpp-with-gradle.md]: Updated with I_n earlier Gradle versions starting from 5.3_

File: pages/docs/reference/building-mpp-with-gradle.md (hunk 3 above)
Comment (katia-energizer) [reply pages/docs/reference/building-mpp-with-gradle.md]: Agree. Removed.

**Pull request merged**